doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -4,14 +4,39 @@
+* [1.5 Verification methodologies](#1.5)
+*	[1.6 Assumptions & Dependencies](#1.6)
+*	[1.7 Results Collation & Presentation](#1.7)
+*	[1.8 Governance](#1.8)
+* [x.x Resources & References](#x.x)
+
+## Synopsis
+
+Ensure Reference Implementation of CNTT Reference Model and CNTT Reference Architecture meets industry driven quality assurance standards for compliance, verification and validation.  The OPNFV Verified Program (OVP), by Linux Foundation Networking (LFN), overseen by the Compliance Verification Committee (CVC), will provide tracking and governance for RM/RA verification.
+
+For the purposes of this chapter, NFVI+VNF testing will be performed for **Verification**, **Validations**, ultimately leading to **Certifications** indicating a measured performance of the adherence to, and demonstrated proficiency with, all aspects of software delivery including but no limited to planning, logistics for communication, and testing of installed, or implmented, NFVI.
 
 <a name="1.1"></a>
-## 1.1 Overview
+## 1.1 Introduction
+
+**Chapter Purpose**<br>
+This chapter includes process flow, logistics, and requirements which must be satisfied to ensure Network Function Virtualisation Infrastructure (NFVI) meets the design, feature, and capability expectations of VNF developers promoting both the use and scalability of Software Defined Networking (SDN) capabilities.  Upstream projects will define features/capabilities, test scenarios, and test cases which will be used to augment OVP test harnesses for infrastructure verification purposes.  Existing processes, communication mediums, and related technologies will be utilized where feasible.  Ultimately, test results of certified NFVI+VNF will reduce the amount of time and cost it takes each operator to on-board and maintain vendor provided VNFs.
+
+**Objective**<br>
+Perform NFVI+VNF Verification and Validations using CNTT reference architecture, leveraging the existing OPNFV and CVC Intake and Validation Process to onboard and validate new test projects for NFVI compliance.  Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
 
-To be added once more content is developed.
+**Test Methodology**
+- Manifest Verifications (Termed Compliance by CVC) will ensure the NFVI is compliant, and delivered for testing, with hardware and software profile specifications defined by the Ref Model and Ref Architecture.
+- Empirical Validation with Reference Golden VNFs (Termed Validation by CVC) will ensure the NFVI runs with a set of VNF Families, or Classes, to minic production-like VNF connectivity, for the purposes of interopability checks.
+- Candidate VNF Validation (Termed Validation & Performance by CVC) will ensure complete interoperablity of VNF behaviour on the NFVI leverage VVP/VNFSDK test suites.  Testing ensures VNF can be spun up, modified, or removed, on the target NFVI (aka Interoperability).
```

Comment:
shall we mention with ONAP along VVP/VNFSDK ?

Reply:
@mkr1481 - I'd recommend leaving ONAP out to continue to support our objectives and principles of being orchestrator agnostic.  Granted, we may still use ONAP during validations, but our verification/validation strategies should be independent from any specific orchestrator.  **Let me know if we can close this out.**

Reply:
@mkr1481 - can we close, or Resolve, this conversation?

Reply:
sure. as we mentioned about VVP/VNFSDK,  wanted to make it explicit. otherwise yes, this document could be generic one.  may be we could mention ONAP as example to give the context for VVP/VNFSDK?